airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -64,11 +63,65 @@
         "title": "Username",
         "order": 5
       },
-      "password": {
-        "description": "The password associated with the username.",
-        "type": "string",
-        "airbyte_secret": true,
-        "title": "Password",
+      "credentials": {
+        "title": "Authorization Method",
+        "type": "object",
+        "oneOf": [
+          {
+            "type": "object",
+            "title": "OAuth2.0",
+            "order": 0,
+            "required": ["client_id", "client_secret"],
+            "properties": {
+              "auth_type": {
+                "type": "string",
+                "const": "Client",
+                "enum": ["Client"],
+                "default": "Client",
+                "order": 0
+              },
+              "client_id": {
+                "type": "string",
+                "title": "Client ID",
+                "description": "The Client ID of your Drift developer application.",
+                "airbyte_secret": true
+              },
+              "client_secret": {
+                "type": "string",
+                "title": "Client Secret",
+                "description": "The Client Secret of your Drift developer application.",
+                "airbyte_secret": true
+              },
```

Comment:
Do we need to reuse these properties inside of the source-connector? If so, leave as it is, otherwise, we can safely remove these ones.

Reply:
Yes, it's also required

Reply:
Ok, but do not make it "required", because these are not available from UI as input fields.

Reply:
ok, will update it.
But it already works and had been tested

Reply:
Updated, Thanks